packages/ui-billboard/src/Billboard/index.tsx

```diff
@@ -115,8 +113,7 @@ class Billboard extends Component<BillboardProps> {
   }
 
   renderHero() {
-    if (this.heroIsFunction) {
-      // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
+    if (typeof this.props.hero === 'function') {
```

Comment:
The original `heroIsFunction` getter can be deleted since it is not used anymore.